posthog/clickhouse/migrations/0050_add_channel_definitions.py

```diff
@@ -0,0 +1,12 @@
+from posthog.clickhouse.client.migration_tools import run_sql_with_exceptions
+from posthog.models.channel_type.sql import (
+    CHANNEL_DEFINITION_TABLE_SQL,
+    CHANNEL_DEFINITIONS_DATA_SQL,
+    CHANNEL_DEFINITION_DICTIONARY_SQL,
+)
+
+operations = [
+    run_sql_with_exceptions(CHANNEL_DEFINITION_TABLE_SQL),
+    run_sql_with_exceptions(CHANNEL_DEFINITIONS_DATA_SQL),
```

Comment:
totally a nit - feel free to ignore. You have plural of `DEFINITIONS` and singular of `DEFINITION` here.